src/js/components/Carousel/__tests__/__snapshots__/Carousel-test.js.snap

```diff
@@ -276,8 +276,6 @@ exports[`Carousel basic 1`] = `
   border: none;
   padding: 0;
   text-align: inherit;
-  opacity: 0.3;
```

Comment:
broken?

Reply:
fixed

Reply:
I don't think this is fixed yet. When the Carousel gets to the last image, the "next" button should be disabled. It was previously but is not when I tested this branch via storybook locally.